keylime/cloud_verifier_common.py

```diff
@@ -270,6 +270,17 @@ def process_get_status(agent):
         al_len = len(allowlist['allowlist'])
     else:
         al_len = 0
+
+    try :
+        mb_refstate = ast.literal_eval(agent.mb_refstate)
+    except Exception as e:
+        logger.warning(f'Non-fatal problem ocurred while attempting to evaluate agent attribute "mb_refstate" ({e.args}). Will just consider the value of this attribute to be "None"')
```

Comment:
I know we're dealing with an AST, and language errors are hard, but is it possible to include more information here? I can imagine trying to debug this being a pain. Maybe we could put some details from the exception in a debug log statement?

Reply:
Would something like `str(agent.mb_refstate)` be enough for a `debug` statement? I am asking because the `warning` already brings the args from the exception

Reply:
Yeah, I think if we have the exception args in the warning and the refstate in a debug (could even put the debug in the `try`) that should be good enough to start.